content/docs/tutorials/converting/resolving-errors.md

```diff
@@ -124,7 +126,7 @@ The problem is that this is an external stylesheet reference. In AMP, to keep th
 
 So, let's resolve the error:
 
-1.  **Remove** the `<link>` tag in the `<head>` and replace it with an inline `<style amp-custom></style>` tag. The `amp-custom` attribute on the style tag is mandatory.
+1.  **Remove** the stylesheet `<link>` tag in the `<head>` and replace it with an inline `<style amp-custom></style>` tag. The `amp-custom` attribute on the style tag is mandatory.
```

Comment:
"remove the `<link>` tag pointing to the stylesheet"

Reply:
Done.